org.eclipse.jdt.text.tests/src/org/eclipse/jdt/text/tests/codemining/ParameterNamesCodeMiningTest.java

```diff
@@ -124,6 +124,47 @@ public class Foo {
 		assertEquals(3, fParameterNameCodeMiningProvider.provideCodeMinings(viewer, new NullProgressMonitor()).get().size());
 	}
 
+	@Test
+	public void testRecordClasses() throws Exception {
```

Comment:
Rename it and move it closer to the already existing test called `testRecordConstructorOK` (they belong together):

```suggestion
	public void testRecordConstructorOneParameter() throws Exception {
```

Reply:
done